app/ledger.js

```diff
@@ -116,8 +117,14 @@ var boot = () => {
 
     if (err.code !== 'ENOENT') console.log('statePath read error: ' + err.toString())
 
-    client = (require('ledger-client'))(null, underscore.extend(clientOptions, { roundtrip: roundtrip }), null)
+    try {
+      client = (require('ledger-client'))(null, underscore.extend({ roundtrip: roundtrip }, clientOptions), null)
+    } catch (ex) {
+      bootP = false
+      return console.log('ledger-client error: ' + ex.toString() + '\n' + ex.stack)
```

Comment:
`Brave-darwin-x64/Brave.app/Contents/Resources/app.asar.unpacked/node_modules/node-anonize2-relic-emscripten/` exists but is empty